### ISSUE TITLE
Example of using MultilayerPerceptron?

### ISSUE THREAD

**Issue by hanlianlu** [aigamedev/scikit-neuralnetwork#173]
Hi everyone,

Is there a brief example of using MultilayerPerceptron? 
I'm kinda confused with ''When using the multi-layer perceptron, you should initialize a Regressor or a Classifier directly."   Concretely whats the difference between applying MLP(sknn.mlp.MultilayerPerceptron) and normal Layer(sknn.mlp.layer)?
A short Example of building a e.g. 5 layer MLP using this will be nice.

Thanks a lot

**Comment by alexjc** [comment]
Have you checked the documentation and the examples folder?

**Comment by hanlianlu** [comment]
Yes, in Docs there are examples of standard layer combined with Classifier like:
from sknn.mlp import Classifier, Layer 
nn = Classifier( layers=[ Layer("Maxout", units=100, pieces=2), Layer("Softmax")], learning_rate=0.001, n_iter=25)

 , I try to use MLP instead of Layer since it has more available attributes. 

Specifically I don't quite understand the first MLP attributes " layer list" . That will be nice of you giving a short example for all of this.

Thanks a lot

**Comment by alexjc** [comment]
The example you pasted is using a list of layers. It's already documented unless I'm missing something...

**Comment by hanlianlu** [comment]
Maybe I didn't clarify my question:).

I mean, in the example I pasted, it use "standard Layer(from sknn.mlp import Layer)" , now that I want to try "MLP" (from sknn.mlp import MultiplayerPerceptron), what should I modify with respect to the example? :)

**Comment by alexjc** [comment]
You're already using both these things, MultilayerPerceptron uses Layers or it wouldn't work. You also can't use layers alone.

**Comment by alexjc** [comment]
Did this answer your question? Going to close this issue, but if you can clarify then feel free to re-open.
